ui/src/pages/sitemaps/users.xml.tsx

```diff
@@ -0,0 +1,20 @@
+import * as Helpers from '@/helpers';
```

Comment:
Why .xml in the filename? Does that do something different in nextjs?

Reply:
Yeah, this is just to control the eventual path - if it didn't contain ".xml" the eventual path would be /sitemaps/users, so instead we've chosen to be explicit about the file type that's going to be delivered.